application/controllers/SniController.php

```diff
@@ -7,17 +7,33 @@
 use Icinga\Exception\NotFoundError;
 use Icinga\Module\X509\Forms\Config\SniConfigForm;
 use Icinga\Module\X509\SniIniRepository;
-use Icinga\Web\Controller;
-use Icinga\Web\Url;
+use ipl\Web\Compat\CompatController;
+use ipl\Web\Url;
+use ipl\Web\Widget\ButtonLink;
 
-class SniController extends Controller
+class SniController extends CompatController
 {
     /**
      * List all maps
      */
     public function indexAction()
     {
-        $this->view->tabs = $this->Module()->getConfigTabs()->activate('sni');
+        $this->getTabs()->add('jobs', [
+            'title'      => $this->translate('Configure Jobs'),
+            'label'      => $this->translate('Jobs'),
+            'url'        => 'x509/jobs',
+            'baseTarget' => '_main'
+
+        ]);
+        $this->addTitleTab($this->translate('SNI'));
+
+        $this->addControl(
+            (new ButtonLink($this->translate('New SNI Map'), Url::fromPath('x509/sni/new'), 'plus'))
+                ->openInModal()
```

Comment:
Don't open this in a modal. The form is the only content. It's not like with the job config where a specific detail is used.